modules/govuk_unattended_reboot/templates/03_no_reboot.erb

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+LOCK_FILES=$(ls -A <%= @lock_dir %>)
+
+  # If any files exist within the lock directory. The Server will not reboot
+
+  if [[ $LOCK_FILES ]]; then
```

Comment:
Does this the right status code when there are files in the directory compared to when there are no files in the directory? 

```
lauramartin@xenial-test-1:~$ mkdir foo
lauramartin@xenial-test-1:~$ ls -A foo/
lauramartin@xenial-test-1:~$ echo $?
0
lauramartin@xenial-test-1:~$ touch foo/bar
lauramartin@xenial-test-1:~$ ls -A foo/
bar
lauramartin@xenial-test-1:~$ echo $?
0
```